pom.xml

```diff
@@ -106,7 +107,7 @@
     <rpm.maven.plugin.version>2.2.0</rpm.maven.plugin.version>
 
     <!-- Code Coverage -->
-    <jacoco.check.line.coverage>0.50</jacoco.check.line.coverage>
+    <jacoco.check.line.coverage>0.53</jacoco.check.line.coverage>
```

Comment:
nit: in future, try to send unrelated changes like this as separate PRs. It's easier for a reviewer to understand the implications of a small PR that does one thing than a big PR that does several things.

Reply:
I totally agree. Could you elaborate on what should not have been included? This particular line is ratcheting down the required coverage as the PR includes more tests. Some of the tests added could probably have been done separately, but in general, changes to required coverage are not unexpected even when just including relevant tests.

e.g. if I add 10 lines of code and cover them to 100% then the overall project coverage percentage should move up too. In a project like this where coverage has been relatively poor historically, I want to ensure we don't regress when we see that improve so all new contributions have to at least maintain that coverage level. (I wish I could configure it to force them to improve it!)

Just making sure we're on the same page here.

Reply:
Oh, sorry, I was skimming and thought this was adjusting a library version. Cancel that comment.

Maybe would have been nice if you'd sent the Mean->Count fix as a separate PR, but I feel less strongly about that (I don't feel strongly about perfect PR hygiene to begin with).